src/pincode/authentication.test.ts

```diff
@@ -65,7 +67,8 @@ jest.mock('src/utils/aes', () => {
 const loggerErrorSpy = jest.spyOn(Logger, 'error')
 const mockPepper = {
   username: 'some username',
-  password: '01010101010101010101010101010101',
+  password:
```

Comment:
Had to update this and some expected hashes so it matched the length asked via `crypto.randomBytes`